lib/bootloader_pvm.pm

```diff
@@ -123,19 +123,24 @@ sub enter_netboot_parameters {
         record_info("Updated boot path for PPC64LE_GRUB_HTTP defined", $mntpoint);
     }
     my $ntlm_p = get_var('NTLM_AUTH_INSTALL') ? $ntlm_auth::ntlm_proxy : '';
-    if (get_var('AGAMA')) {
-        type_string_slow "linux $mntpoint/linux vga=normal root=live:http://" . get_var('OPENQA_HOSTNAME') . "/assets/iso/" . get_var('ISO');
+    if (is_agama) {
+        type_string_slow "linux $mntpoint/linux root=live:http://" . get_var('OPENQA_HOSTNAME') . "/assets/iso/" . get_var('ISO') . " live.password=$testapi::password";
+        type_string_slow " agama.auto=" . data_url(get_var('AGAMA_AUTO')) if (get_var('AGAMA_AUTO'));
```

Comment:
logic for autoyast was before in specific_bootmenu_params, we implemented there for auto agama.
this would repeat the line and would be missing features like replacement of profile variable.

Reply:
Thanks, I missed the coverage totally.